### PR TITLE
[APM-CI] enable notifications

### PR DESCRIPTION
**Pull request by kuisathaverat**
* enable notifications
* remove withEnvWrapper step

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -47,27 +50,26 @@ pipeline {
...
         HOME = "${env.WORKSPACE}"
         GOPATH = "${env.WORKSPACE}"
       }
-      when { 
+      when {
         beforeAgent true
         expression { return params.intake_ci }
       }
       steps {
-        withEnvWrapper() {
```
Comment (simitt) [inline Jenkinsfile]: Why is this removed?

Comment (kuisathaverat) [reply Jenkinsfile]: this step was created as a workaround until we can use credentials for environment variables like NOTIFY_TO, now we have the credentials created so it is no longer necessary.

**Comment by kuisathaverat** [comment]
the error in the CI is not related to the PR

```
diff --git a/NOTICE.txt b/NOTICE.txt
index 735f3bb..9d4aa52 100644
--- a/NOTICE.txt
+++ b/NOTICE.txt
@@ -1,5 +1,5 @@
 Apm Server
-Copyright 2014-2018 Elasticsearch BV
+Copyright 2014-2019 Elasticsearch BV

 This product includes software developed by The Apache Software 
 Foundation (http://www.apache.org/).
NOTICE.txt: needs update
Makefile:93: recipe for target 'check-full' failed
make: *** [check-full] Error 1
+ cleanup
+ rm -rf /tmp/tmp.UnJIar6oaq
script returned exit code 2
```
